Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -662,7 +662,7 @@ def output(result: Optional[CollectionResult]):
 
 
 if __name__ == '__main__':
-    logger.info('TestCollector v20220811')
+    logger.info('TestCollector v20220811.2')
```

Comment:
what's this change?

Reply:
mostly for debug purposes, to know exactly what TestCollect version is running on each build (with this fix, w/o the other fix etc)

Reply:
Thanks for the clarification!